tests/helpers/pactHelper.js

```diff
@@ -55,7 +55,7 @@ const shareResponseOcsData = function (node, shareType, id, permissions, fileTar
     .appendElement('uid_file_owner', '', MatchersV3.string(config.adminUsername))
     .appendElement('displayname_file_owner', '', MatchersV3.string(config.adminUsername))
     .appendElement('path', '', MatchersV3.equal(fileTarget))
-    .appendElement('file_target', '', MatchersV3.equal(fileTarget))
+    .appendElement('file_target', '', MatchersV3.regex(`(\\/Shares)?\\${fileTarget}`, fileTarget))
```

Comment:
the issues with that is, if the fileTarget has special char that have some meaning in regex it does not work 
e.g.
`/test space and + and #.txt` does not match `(\/Shares)?\/test space and + and #.txt`
for that the regex should be `(\/Shares)?\/test space and \+ and #\.txt`

Reply:
so would need to adjust the regex or maybe `MatchersV3.includes` will be enough
includes | value: string | Value that must include the example value as a substring.

Reply:
Ohh, I was wondering why all the changed passed locally but not in CI.
It looks like I have not touched the test that uses this interaction.